spec/requests/schools/ects/change_lead_provider_wizard_spec.rb

```diff
@@ -171,6 +171,42 @@
           expect(response).to redirect_to(path_for_step("confirmation"))
         end
       end
+
+      context "when the ECT is school-led" do
+        let(:training_period) do
+          FactoryBot.create(
+            :training_period,
+            :ongoing,
+            :school_led,
+            ect_at_school_period:,
+            started_on: ect_at_school_period.started_on
+          )
+        end
+
+        before do
+          sign_in_as(:school_user, school:)
+        end
```

Comment:
I don't think this is needed, we're already doing it ⤴️ [here](https://github.com/DFE-Digital/register-early-career-teachers-public/pull/1633/files#diff-95edfd5f2735697eef9560c527d082d1b47742ce3675e4e2a51c1451f348f5eaL108)